docs/quickstart.rst

```diff
@@ -5,28 +5,117 @@ Quickstart
 
 Installing & Running Docker
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-For this tutorial, we'll run Docker using the Docker client.  If you are interested in information on using Docker Compose to run the images, :ref:`we have docs for that too <quickstart_compose>`.
+For this tutorial, you can run Docker using the Docker client or Docker Compose:
 
-To get started, you'll need to first `install Docker and get it running <https://docs.docker.com/engine/installation/>`_.  The CP Docker Images require Docker version 1.11 or greater.
+* `Getting Started with Docker Compose`_
+* `Getting Started with Docker Client`_
 
-Running Confluent Platform on Docker
+
+.. _quickstart_compose:
+
+Getting Started with Docker Compose
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Docker Compose is a powerful tool that enables you to launch multiple docker images in a coordinated fashion.  It is ideal for platforms like Confluent.  Before you get started, you will need to install both the core `Docker Engine <https://docs.docker.com/engine/installation/>`_ and `Docker Compose <https://docs.docker.com/compose/install/>`_.  Once you've done that, you can follow the steps below to start up the Confluent Platform services.
+
+1. Create and configure the Docker Machine {if you don't want to reuse your Docker Host from the Quickstart above} (OS X only).
+
+  .. sourcecode:: bash
+
+    docker-machine create --driver virtualbox --virtualbox-memory 6000 confluent
+
+  Next, configure your terminal window to attach it to your new Docker Machine:
+
+  .. sourcecode:: bash
+
+    eval $(docker-machine env confluent)
+
+2. Clone the CP Docker Images Github Repository.
+
+  .. sourcecode:: bash
+
+    git clone https://github.com/confluentinc/cp-docker-images
+
+  An example Docker Compose file is included that will start up ZooKeeper and Kafka. Navigate to ``cp-docker-images/examples/kafka-single-node``, where it is located.  Alternatively, you can download the file directly from https://github.com/confluentinc/cp-docker-images/raw/master/examples/kafka-single-node/docker-compose.yml 
+
+  .. sourcecode:: bash
+    cd cp-docker-images/examples/kafka-single-node
+
+
+3. Start ZooKeeper and Kafka using Docker Compose ``create`` and ``start`` commands.  You'll run these commands from the directory containing the docker-compose.yml file.
+
+   .. sourcecode:: bash
+
+       docker-compose create
```

Comment:
I'm not familiar with how create/start are different from up.  In other places, e.g. in KSQL docs, we told people to do `docker-compose up -d`.  When to use which one?

Reply:
i *think* docker up does both create and start, per https://docs.docker.com/compose/reference/up/